debezium-bom/pom.xml

```diff
@@ -181,6 +181,12 @@
                 <version>${version.com.google.protobuf}</version>
             </dependency>
 
+            <dependency>
+                <groupId>com.google.protobuf</groupId>
+                <artifactId>protobuf-java-util</artifactId>
+                <version>${version.com.google.protobuf}</version>
+            </dependency>
+
```

Comment:
Thanks @janjwerner-confluent for the CVE fix.
Any reason we are adding this particular dependency?

Reply:
Thanks. 
LGTM.